Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/Pass.h

```diff
@@ -222,6 +223,13 @@ namespace AZ
             //! Enables/Disables PipelineStatistics queries for this pass
             virtual void SetPipelineStatisticsQueryEnabled(bool enable);
 
+            //! Readback an attachment attached to the specified slot name
+            //! @param readback The AttachmentReadback object which is used for readback. Its callback function will be called when readback is finished.
+            //! @param attachment The pass attachment to be readback. It need to be one of this pass's attachment
```

Comment:
the param is slotName here (although not sure if it is obviously synonymous to attachment to other people)